deployment/openshift-play-scala-grpc-example.yml

```diff
@@ -0,0 +1,87 @@
+---
+apiVersion: "apps/v1beta2"
+kind: Deployment
+metadata:
+  name: "play-scala-grpc-example-v1-0-snapshot"
+  labels:
+    appName: "play-scala-grpc-example"
+    appNameVersion: "play-scala-grpc-example-v1-0-snapshot"
+spec:
+  replicas: 2
+  selector:
+    matchLabels:
+      appNameVersion: "play-scala-grpc-example-v1-0-snapshot"
+  template:
+    metadata:
+      labels:
+        appName: "play-scala-grpc-example"
+        appNameVersion: "play-scala-grpc-example-v1-0-snapshot"
+    spec:
+      restartPolicy: Always
+      containers:
+        - name: "play-scala-grpc-example"
+          image: "docker-registry-default.centralpark.lightbend.com/play-scala-grpc-example/play-scala-grpc-example:1.0-SNAPSHOT"
```

Comment:
The image coordinates are _the_ only relevant difference in `openshift.yaml` when compared to `kubernetes.yaml` (other than routes vs ingress).

Reply:
I'd like these coordinates to _not_ be centralpark-specific.